tests/test_core/test_new_moltype.py

```diff
@@ -192,3 +192,20 @@ def test_get_degenerate_positions(data_type, moltype):
     got = moltype.get_degenerate_positions(seq)
     expect = numpy.array([], dtype=bool)
     assert numpy.array_equal(got, expect)
+
+
+@pytest.mark.parametrize(
+    "moltype",
+    (
+        new_moltype.ASCII,
+        new_moltype.DNA,
+        new_moltype.RNA,
+        new_moltype.PROTEIN,
+        new_moltype.PROTEIN_WITH_STOP,
+    ),
+)
+def test_gaps(moltype):
+    # TODO: fred, bytes
```

Comment:
I don't quite understand how to test for `new_moltype.BYTES` e.g. `MolType(range(0, 255)))`

Reply:
BYTES does defines both gap and missing as None. But looking at the original implementation, it just uses the defaults. So I think we should modify the current BYTES definition to reflect that of the original.

Reply:
Thanks - will add to next PR